nautobot/extras/datasources/git.py

```diff
@@ -364,10 +492,8 @@ def update_git_config_contexts(repository_record, job_result):
 def import_config_context(context_data, repository_record, job_result, logger):
     """
     Parse a given dictionary of data to create/update a ConfigContext record.
-
```

Comment:
Please revert the whitespace changes here and below. Python docstring convention (https://www.python.org/dev/peps/pep-0257/) is to have a blank line following the first line of a multi-line docstring.